### ISSUE TITLE
Track Changes Feedback Not Announcing In MS Word

### ISSUE THREAD

**Issue by FoxMachina** [nvaccess/nvda#12128]
### Steps to reproduce:
Open a document in MS Word, activate track changes. When making edits to the document, the screen reader no longer says inserted and deleted. All the insertions and deletions do not automatically read as before. Under preferences, I have notes and comments and editors revisions checked under document formatting. 

### Actual behavior:
When making tracked changes, NVDA does not read them at all. Text appears to be normal, but if right clicked, options for accept/deny insertion/deletion does appear as an option.

### Expected behavior:
When reading a document, NVDA should announce inserted and deleted along with the relevant portions that were added/deleted.

#### NVDA installed/portable/running from source:
Installed locally

#### NVDA version:
Tested with 220.4 and Alpha 21810, db664bef

#### Windows version:
Windows 10 Enterprise

#### Name and version of other software in use when reproducing the issue:
MS Office 365

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Same behavior in 2020.3 and 2020.4

#### If add-ons are disabled, is your problem still occurring?
Yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No

**Comment by Qchristensen** [comment]
@FoxMachina can you still replicate this?  I just tried it, and I found:

- When typing, whether you are inserting text, or deleting text is not reported while typing.
- While navigating, NVDA correctly reports whether text is inserted, deleted, or not inserted (when moving between inserted and original text).
- This behaviour is the same regardless of the state of "Use UI Automation to access Microsoft Word controls" in NVDA's advanced settings.

Tested using NVDA 2020.4
on Windows 10 (64-bit) Version: 2009, Build: 19042
running Office 365 (64-bit) Version: 16.0.13901.20300
